test/karma.conf.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (config) {
             'app/lib/command.js',
             'app/lib/condition.js',
             'app/lib/manifest-reader.js',
+            'app/lib/boolean-phrase-parser.js',
```

Comment:
done. But shouldn't this be solved with a module loader. I saw you are using $script. can it solve this issue?